examples/webgl_lights_spotlight.html

```diff
@@ -232,6 +233,15 @@
 
 				} );
 
+
+				gui.add( params, 'enable shadows' ).onChange( function ( val ) {
+
+					renderer.shadowMap.enabled = val
+					material.needsUpdate = true;
```

Comment:
This flag is necessary, otherwise, the change is only visible when the spotlightmap is changed (some texture to none, or no texture to some)